be/src/storage/lake/lake_persistent_index.h

```diff
@@ -69,9 +78,24 @@ class LakePersistentIndex : public PersistentIndex {
 
     Status major_compact(int64_t min_retain_version);
 
+private:
+    void flush_to_immutable_memtable();
+
+    bool is_memtable_full();
+
+    Status get_from_immutable_memtable(size_t n, const Slice* keys, IndexValue* values,
```

Comment:
comments should be added to these functions and parameters, what are [in] parameters, what are [out] parameters, and how does the function modifies these parameters.